docarray/documents/text.py

```diff
@@ -102,10 +104,26 @@ class MultiModalDoc(BaseDoc):
 
     """
 
-    text: Optional[str]
-    url: Optional[TextUrl]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[bytes]
+    text: Optional[str] = Field(
+        description='The text content stored in the document',
+        example='This is an example text content of the document',
+    )
+    url: Optional[TextUrl] = Field(
+        description='''The url of the text content. When text content is too long
+    to be stored inline or in a file, the remote url can be used to load the
+        text content''',
+        example='https://www.w3.org/History/19921103-hypertext/hypertext/README.html',
+    )
+    embedding: Optional[AnyEmbedding] = Field(
+        description='''Embedding field is used to store tensor objects of type
+        Tensorflow, PyTorch, and NumPy''',
```

Comment:
Also Jax is supported

Reply:
Ok will add the mention of Jax tensor object type

Reply:
I do not see the Jax right?